commandsContent.js

```diff
@@ -0,0 +1,73 @@
+const metaInfo = {
+    color: "#cc9543",
+}
```

Comment:
Can we move the `color` field to the config file at the root? I say this because we'll need color elsewhere in the codebase as well.